indra/llimage/llimagedxt.cpp

```diff
@@ -207,10 +207,7 @@ bool LLImageDXT::updateData()
 		height = header->maxheight;
 	}
 
-	if (data_size < mHeaderSize)
-	{
-		LL_ERRS() << "LLImageDXT: not enough data" << LL_ENDL;
-	}
+	llassert_message_return_false(data_size >= mHeaderSize, "LLImageDXT: not enough data")
```

Comment:
Disagree. If this actually happens, memory is likely corrupted, viewer should quit, not keep going.